SYCL/ESIMD/api/simd_view_copy_move_assign.cpp

```diff
@@ -114,6 +114,13 @@ int main(void) {
   passed &= test(q, "move assignment operator",
                  [](auto &va_view, auto &vb_view)
                      SYCL_ESIMD_FUNCTION { va_view = std::move(vb_view); });
+  // construct complete view of a vector.
+  passed &= test(q, "constructor from simd",
+                 [](auto &va_view, auto &vb_view) SYCL_ESIMD_FUNCTION {
+                   simd<int, 4> vb = vb_view;
```

Comment:
Please add a test case with VL=1

Reply:
I added the test case for VL=1: 0d2eb25e853ada96e9b5a5e4b920b9cb5cd6b5e9.

During adding that I discovered that 'simd v = other_simd.select<4,1>(0)' does not work and that may be pretty inconvenient. That is also added here: https://github.com/intel/llvm/pull/4892